devel/ci/integration/tests/fixtures/greenwave.py

```diff
@@ -24,19 +24,16 @@
 
 
 @pytest.fixture(scope="session")
-def greenwave_container(docker_backend, docker_network, db_container):
+def greenwave_container(docker_backend, docker_network):
     """Fixture preparing and yielding a Greenwave container.
 
     Args:
         docker_backend (conu.DockerBackend): The Docker backend (fixture).
         docker_network (str): The Docker network ID (fixture).
-        db_container(conu.DockerContainer): The PostgreSQL container (fixture).
 
     Yields:
         conu.DockerContainer: The Greenwave container.
     """
-    # Prepare the database
-    make_db_and_user(db_container, "greenwave")
```

Comment:
Ah interesting, I didn't realize that Greenwave didn't use a DB. I went and took a peek at their settings and I see that they are not. I also see that we don't download a Greenwave db dump.